pydoctor/sphinx.py

```diff
@@ -134,6 +136,38 @@ def getLink(self, name: str) -> Optional[str]:
         return f'{base_url}/{relative_link}'
 
 
+def _parseInventoryLine(line: str) -> Tuple[str, str, int, str, str]:
```

Comment:
```suggestion
def _parseSphinxInventoryLine(line: str) -> Tuple[str, str, int, str, str]:
```

Reply:
Since this is a private function in a module named `pydoctor.sphinx`, I think it's clear enough from the context that it's a Sphinx inventory we're parsing.